styles/main.css

```diff
@@ -253,6 +315,109 @@ header {
   margin-right: 10px;
 }
 
+@media screen and (min-width: 768px) {
+  .card__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .work {
+    justify-content: center;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    height: 100%;
+    border: 1px solid #d0d9d4;
+    border-radius: 4rem;
+    margin: 2%;
+    overflow: hidden;
+  }
+
+  .card:nth-child(2n) {
+    flex-direction: row-reverse;
+  }
+
+  .card__imgbox {
+    height: 100%;
+    width: 50%;
+  }
```

Comment:
- Hey, let's try to improve our design by adjusting the styles on the image as it goes out of the card.

*Your card*
![Screenshot from 2023-05-04 18-44-12](https://user-images.githubusercontent.com/59397159/236287637-edb7616f-4b68-4050-aa1d-d3324ef64064.png)

*your design*
![image](https://user-images.githubusercontent.com/59397159/236287913-c9d16375-7e42-4c17-a2da-1fffcbc0e43b.png)